README.md

```diff
@@ -1,6 +1,20 @@
 # std-toast
 This document scopes out a web platform API for a 'toast' pop-up notification.
 
+## What is a "Toast" Pop-Up Notification?
+
+_"Toasts are pretty common in UX design; refers to popup notifications which typically appear at the bottom of the screen (like a piece of toast in a toaster)."_ (credit: [Kyle Decker](https://twitter.com/kybradeck/status/1139006173762531328))
+
+Sneha Munot provides a nice definition in her [uxplanet.org article](https://uxplanet.org/toast-notification-or-dialog-box-ae32ad53106d), where she compares toasts to dialog boxes.
+
+A toast...
+> ...is a small message that shows up in a box at the bottom of the screen and disappears on its own after few seconds. It is a simple feedback about an operation in which current activity remains visible and interactive. It basically is to inform the user of something that is not critical and that does not require specific attention and does not prevent the user from using the app device.  For example; on gmail when a mail is send you receive a feedback of “Sending message…” written in the form of toast message.
+
+Furthermore, Ben Brocka has a nice, concise definition, in his [ux.stackexchange.com response](https://ux.stackexchange.com/a/12000), _"A Toast is a non modal, unobtrusive window element used to display brief, auto-expiring windows of information to a user."_, which adds the distinguishing detail of a toast's being **auto-expiring**.
+
+In the absence of browser-intrinsic toasts, the current state of affairs is that libraries like Bootstrap include [toast features like this one](https://getbootstrap.com/docs/4.2/components/toasts/).  Additionally, here is the Toast Study Group's [list of 3rd party libraries](https://github.com/jackbsteinberg/std-toast/tree/master/study-group) that includes an awesome table of comparison.
```

Comment:
calling your own study "awesome" seems a bit pretentious, and it also doesn't seem like including that words adds anything to the sentence? (it's also not clear to me who or what the "Toast Study Group" is)

Reply:
Thanks for pointing this out. Notice that this was written by a third-party contributor---thanks @monico-moreno for the compliment! But yeah, I'll touch it up before merging.

Reply:
You're welcome, @domenic.   Good catch, @mkruisselbrink.  I will, however, leave the question as to what the Toast Study Group is to the experts.  Perhaps a link to a description segment or page? 🤔